README.rst

```diff
@@ -20,21 +20,25 @@
 .. _Depsy: http://depsy.org/package/python/matplotlib
 
 ##########
-matplotlib
+Matplotlib
 ##########
 
-matplotlib is a Python 2D plotting library which produces publication-quality
-figures in a variety of hardcopy formats and interactive
-environments across platforms. matplotlib can be used in Python
-scripts, the Python and IPython shell (ala MATLAB or Mathematica), web
-application servers, and various graphical user interface toolkits.
+Matplotlib is a Python 2D plotting library which produces publication-quality
+figures in a variety of hardcopy formats and interactive environments across
+platforms. Matplotlib can be used in Python scripts, the Python and IPython
+shell (ala MATLAB or Mathematica), web application servers, and various
```

Comment:
you know how to spell "à la" (it's correct even in English, see https://en.wiktionary.org/wiki/%C3%A0_la)